app/views/tables.html

```diff
@@ -1,9 +1,13 @@
 <div>
   <div ng-class="$root.showSideList ? '' : 'hide-sidebar-list'">
   	<div class="page-container" ng-if="empty">
-      <h1 class="cr-page-header">Tables</h1>
+      <h1 class="cr-page-header">{{ 'TABLE.TITLE' | translate }}</h1>
       <div class="cr-panel-block">
-        There are no tables created yet.
+        <p>
+          <b>There are no tables created yet.</b>
```

Comment:
Would prefer to wrap these in their on `<p></p>` tags rather than using `<br /><br />`, but its just a personal preference. I think the text in the Help tab does it this way.